lib/router.js

```diff
@@ -24,19 +24,11 @@ var SwaggerRouter = swaggerRouter.Router;
  *      - {object} operations, Object mapping operationId -> handler
  */
 function ApiScope(init) {
-    if (!init) {
-        init = {};
-    }
-    this.prefixPath = init.prefixPath || '';
-
-    var specRoot = this.specRoot = init.specRoot || {};
-    if (!specRoot.paths) { specRoot.paths = {}; }
-    if (!specRoot.definitions) { specRoot.definitions = {}; }
-    if (!specRoot.securityDefinitions) { specRoot.securityDefinitions = {}; }
-    if (specRoot.basePath === undefined) { specRoot.basePath  = this.prefixPath; }
-
+    init = init || {};
+    this.specRoot = init.specRoot;
```

Comment:
`init` is not mandatory :) We only require that by the time `specRoot` will be used it will already be set up, so I can't really verify that here.